### ISSUE TITLE
[Proposal] Переработка сканеров тела.

### ISSUE THREAD

**Issue by FBitSafe** [TauCetiStation/TauCetiClassic#6867]
#### Подробное описание проблемы

По моему мнению, сканер является слишком имбовым и не интересным, т.к. всё, что нужно сделать для диагностики пациента - это засунуть его в сканер.

#### Предложения

1. **СКАНЕР**
Сканер нужно урезать в возможностях и добавить их через улучшение машинерии. По стандарту, сканер может делать самые базовые сканы - рентген конечностей, указывать на посторонние предметы в теле, показывать уровень крови. Сделать [GUIDE to Medicine](https://wiki.taucetistation.org/Guide_to_Medicine#.D0.97.D0.B0.D1.80.D0.B0.D0.B6.D0.B5.D0.BD.D0.B8.D0.B5_.D0.B8.D0.BD.D1.84.D0.B5.D0.BA.D1.86.D0.B8.D0.B5.D0.B9) великой снова

2. **ДИАГНОСТИКА**
Добавить возможность анализировать кровь пациента с помощью _**особого**_ прибора, который будет указывать на проблемы с почками, апендицитом или печенью.

#### Что происходит на самом деле

Игра за хирурга превращается в рутину и работу по списочку, ведь сканер скажет в чём проблема, а ты сам ничего не диагностируешь, с таким же успехом можно автохирурга портировать из CM. Фонарик выкидывается в начале раунда или, если не выкидывается, то почти никогда не используется. Стетоскопы просто для антуража лежат в меде.

_я буду обновлять ишью, если найду, что ещё можно предложить по этой теме, а пока можете прокомментировать моё нытьё или поныть вместе_

**Comment by Motzord** [comment]
Итак мы имеем  и так медбей малогеев с 3,5 инвалидами-хирургами, которые два часа лечат перелом потому что не могут сбросить брут и выдернуть осколок из твоего тела
### (Конечно  иногда бывают моменты продуктивного медотсека где все как на заводе)

ты значит предлагаешь, сделать так, чтоб медотсек даже не мог нормально диагностировать заболевание пациента и в красный код, хирург вместо оперирования миллиона раненых, предлагаешь хирургам полчаса облапывать пациентов чтобы узнать че у него вообще болит.
(А мы знаем какой наплыв пациентов под конец раунда, с оторванными руками, отрубленными головами, пулями в жопе и просто мимокрокодилов которым срочно нужен гиперзин-антитоксин-перидаксон)

**Comment by FBitSafe** [comment]
> Итак мы имеем медбей малогеев с 3,5 инвалидами-хирургами, которые два часа лечат перелом потому что не могут сбросить брут и выдернуть осколок из твоего тела

После апдейта вернуться, заебавшиеся от однообразия, спецы своего дела, так и мне самому хотелось бы вернуться в качестве хирурга.

> ты значит предлагаешь, сделать так, чтоб медотсек даже не мог нормально диагностировать заболевание пациента и в красный код, хирург вместо оперирования миллиона раненых, будет полчаса облапывать пациентов чтобы узнать че у него вообще болит.

Зачем облапывать пациента полчаса? Просто разделить функционал сканера по приспособам. Протыкал человека инструментами, основываясь на его показаниях, создал список и понёс, а если забыл что у него болело, то протыкал человека уже на столе и тащить его в сканер не придётся.

В принципе сканер сраная херьня, он делает Guide to Medicine и его таблицу бессмысленной, пришёл пациент такой и говорит:
-мне хуёга!
**Всё**, показания пациента тут закончились, его ложат в сканер, а потом в операционную.

> (А мы знаем какой наплыв пациентов под конец раунда, с оторванными руками, отрубленными головами, пулями в жопе и просто мимокрокодилов которым срочно нужен гиперзин-антитоксин-перидаксон)

Если говорить о наплыве под конец раунда, то тут всё просто, у РнД с большой вероятностью будут нужные ресурсы и исследования и мед будет работать как обычно

Можно ввести дополнительные методы диагностики, а урезать функционал сканера не сразу

**Comment by Motzord** [comment]
> > Итак мы имеем медбей малогеев с 3,5 инвалидами-хирургами, которые два часа лечат перелом потому что не могут сбросить брут и выдернуть осколок из твоего тела
> 
> После апдейта вернуться, заебавшиеся от однообразия, спецы своего дела, так и мне самому хотелось бы вернуться в качестве хирурга.
> 
> > ты значит предлагаешь, сделать так, чтоб медотсек даже не мог нормально диагностировать заболевание пациента и в красный код, хирург вместо оперирования миллиона раненых, будет полчаса облапывать пациентов чтобы узнать че у него вообще болит.
> 
> Зачем облапывать пациента полчаса? Просто разделить функционал сканера по приспособам. Протыкал человека инструментами, основываясь на его показаниях, создал список и понёс, а если забыл что у него болело, то протыкал человека уже на столе и тащить его в сканер не придётся.
> 
> В принципе сканер сраная херьня, он делает Guide to Medicine и его таблицу бессмысленной, пришёл пациент такой и говорит:
> -мне хуёга!
> **Всё**, показания пациента тут закончились, его ложат в сканер, а потом в операционную.
> 
> > (А мы знаем какой наплыв пациентов под конец раунда, с оторванными руками, отрубленными головами, пулями в жопе и просто мимокрокодилов которым срочно нужен гиперзин-антитоксин-перидаксон)
> 
> Если говорить о наплыве под конец раунда, то тут всё просто, у РнД с большой вероятностью будут нужные ресурсы и исследования и мед будет работать как обычно
> 
> Можно ввести дополнительные методы диагностики, а урезать функционал сканера не сразу

тогда требуется оптимизировать медотсек, чтобы он был рассчитан на наплыв пациентов. Будь у рнд хоть ультраблюспейс детали, прогрессбар на собирание конструктора из очередного офицера это не уберет. (А на одну операцию на органы, уходит по несколько минут)

**Comment by FBitSafe** [comment]
> тогда требуется оптимизировать медотсек, чтобы он был рассчитан на наплыв пациентов. Будь у рнд хоть ультраблюспейс детали, прогрессбар на собирание конструктора из очередного офицера это не уберет. (А на одну операцию на органы, уходит по несколько минут)

Я писал этот ишью с упором на разнообразие и больший ролевой отыгрыш. Хочу, чтобы пациент был вынужден сказать о своей проблеме, а не пихать его в сканер, как только он явился в мед. Хотя, разве плохо, что мед не может справиться со всеми пациентами? В таком случае, медики должны будут давать приоритет разным больным.

**Comment by kiVts** [comment]
> , его ложат в сканер, а потом в операционную.

Нет, его несут в операционую а потом не двигая жопой сначала в сканер а потом на стол

**Comment by muWander** [comment]
В последнее время связанные с медициной изменения были направлены на ускорение геймплея, не вижу смысла это менять. Повозиться с пациентом можно и без этого, хотя бы медзаписи обновить.
